float64 flightLegLen
float64 retractDiff
float64 flightP
float64 flightD
float64 stanceP
float64 stanceD
float64 hipP
float64 hipD

# 0 = abs, 1 = inc
int8    stancePSrc
int8    stanceDSrc

# 0 = left, 1 = right, 2 = both, 3 = alternating
int8    legMode

# 0 = disable, 1 = enable
int8    lockLeg
